tst/EnergyPlus/api/TestDataTransfer.py

```diff
@@ -32,6 +32,8 @@ def time_step_handler(state):
     print("Reading outdoor temp via getVariable, value is: %s" % oa_temp)
     dp_temp = api.exchange.get_variable_value(state, outdoor_dew_point_sensor)
     print("Actuated Dew Point temp value is: %s" % dp_temp)
+    tomorrow_db = api.exchange.tomorrow_weather_outdoor_dry_bulb_at_time(state, 3, 2)
```

Comment:
This demonstrates calling one of the new weather worker functions through the C API in a callback scenario.